.github/workflows/js-css-lint-test.yml

```diff
@@ -74,3 +74,12 @@ jobs:
         run: npm run lint:js
       - name: Jest Tests
         run: npm run test:js
+      - name: Jest Tests (Retry Failures 1)
+        run: npm run test:js -- --onlyFailures
+        if: failure()
```

Comment:
It looks like we may need to be more specific about the job that failed here so that it doesn't run when ANY previous step fails, but it shouldn't be a significant change.
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#failure